components/downloadModal.tsx

```diff
@@ -54,71 +54,79 @@ const DownloadModal = () => {
                 </button>
               </div>
             </div>
-            <div className="flex flex-row m-4">
-              <div className="flex flex-col text-center text-xl w-1/4 ">
-                <div className="my-3">OS</div>
-                <div className="my-3">対応モード</div>
-              </div>
-              <div className="flex flex-col text-center text-xl w-3/4">
-                <div className="flex flex-row my-1.5 mx-auto">
-                  <button
-                    className={
-                      osType === 'win'
-                        ? selectedButtonClass
-                        : unselectedButtonClass
-                    }
-                    onClick={() => setOsType('win')}
-                  >
-                    Windows
-                  </button>
-                  <button
-                    className={
-                      osType === 'mac'
-                        ? selectedButtonClass
-                        : unselectedButtonClass
-                    }
-                    onClick={() => {
-                      setOsType('mac')
-                      setSupportMode('cpu')
-                    }}
-                  >
-                    Mac
-                  </button>
-                  <button
-                    className={
-                      osType === 'linux'
-                        ? selectedButtonClass
-                        : unselectedButtonClass
-                    }
-                    onClick={() => setOsType('linux')}
-                  >
-                    Linux
-                  </button>
+            <div className="m-4">
+              <div className="flex flex-row my-1.5">
+                <div className="flex text-xl w-full">
+                  <div className="my-3 w-1/4 text-center">OS</div>
```

Comment:
```suggestion
                  <div className="flex items-center my-3 w-1/4 justify-center">
                    OS
                  </div>
```
せっかくなのでここもセンタリングするようにしましょう！

Reply:
Suggestionありがとうございます！

よく分かっていなかったのですが、ぶら下がっている子要素をセンタリングしたいときにはitems-centerおよびjustify-centerを、テキストをセンタリングしたい時にtext-centerを使うんですね、なるほど

ref:
- https://tailwindcss.com/docs/justify-content#center
- https://tailwindcss.com/docs/align-items#center
- https://tailwindcss.com/docs/text-align#setting-the-text-alignment

なので、今回だと全てを括っている[ここ](https://github.com/SHAREVOX/SHAREVOX.github.io/blob/fix/download-modal/components/downloadModal.tsx#L57)に付与するのが適切でしたね